packages/admin-next/dashboard/src/v2-routes/promotions/promotion-list/loader.ts

```diff
@@ -0,0 +1,25 @@
+import { AdminPromotionsListRes } from "@medusajs/medusa"
+import { Response } from "@medusajs/medusa-js"
+import { QueryClient } from "@tanstack/react-query"
+
+import { adminPromotionKeys } from "../../../lib/api-v2"
+import { medusa, queryClient } from "../../../lib/medusa"
+
+const promotionsListQuery = () => ({
+  queryKey: adminPromotionKeys.list({ limit: 20, offset: 0 }),
+  // Question: whats the v2 equivalenet of this?
```

Comment:
not sure if there is any agreed pattern atm but Oli started defining hooks under `lib/api-v2/*` [e.g. here](https://github.com/medusajs/medusa/pull/6821/files#diff-026e3394090d3730212762d5f3d113bacae5dba4c27a2df676f00a7539a9b2c0R53)